.woloxci/config.yml

```diff
@@ -0,0 +1,21 @@
+config:
+  dockerfile: .woloxci/Dockerfile
+  project_name: express-js-bootstrap
+
+services:
+  - postgres
+
+steps:
+  copy_node_modules:
+    - cp -r $NODE_PATH/ ./
+  migrate_db:
+    - node_modules/.bin/sequelize db:migrate
+  lint:
+    - npm run lint
+  test:
+    - npm run test
```

Comment:
Just a little detail, in the case of test you can avoid the run keyword, you can call `npm test`